### DISCUSSION TITLE
php loading question

### DISCUSSION THREAD

**Discussion by akagumo** [Homebrew/discussions#5737] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.4.5
ORIGIN: https://github.com/Homebrew/brew
HEAD: 254bf3fe9d8fa2e1b2fb55dbcf535b2d870180c4
Last commit: 5 days ago
Core tap JSON: 16 Nov 07:20 UTC
Core cask tap JSON: 16 Nov 07:20 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 12
Homebrew Ruby: 3.3.6 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.6/bin/ruby
CPU: dodeca-core 64-bit kabylake
Clang: 15.0.0 build 1500
Git: 2.47.0 => /usr/local/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 14.7.1-x86_64
CLT: 16.1.0.0.1.1729049160
Xcode: 15.2
```

### Output of `brew doctor`

```shell
Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  php@8.0

Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/IKStealthPedalAPI.dylib
  /usr/local/lib/IKStompIO1API.dylib
  /usr/local/lib/Spyder.dylib
  /usr/local/lib/Spyder3.dylib
  /usr/local/lib/Spyder5.dylib
  /usr/local/lib/dccmtr.dylib
  /usr/local/lib/libCIEHandler.dylib
  /usr/local/lib/libCalibrationHandler.dylib
  /usr/local/lib/libCompression.dylib
  /usr/local/lib/libDeviceManager.dylib
  /usr/local/lib/libErrorHandler.dylib
  /usr/local/lib/libICMManager.dylib
  /usr/local/lib/libMCCAppManager.dylib
  /usr/local/lib/libMCCEngineWrapper.dylib
  /usr/local/lib/libMonitorDDCCISDK.dylib
  /usr/local/lib/libProtocolEngine.dylib
  /usr/local/lib/libQM2Lib.dylib
  /usr/local/lib/libftd2xx.1.4.24.dylib
  /usr/local/lib/libndi.4.dylib
  /usr/local/lib/libndi.dylib

Warning: Your Xcode (15.2) is outdated.
Please update to Xcode 15.4 (or delete it).
Xcode can be updated from the App Store.
```

### Description of issue

is there supposed to be two php in the LaunchAgents folder?
homebrew.mxcl.php.plist
homebrew.mxcl.php@8.1.plist

**Comment by SMillerDev** [comment]
If you started php and php@8.1, yes.

**Comment by akagumo** [comment]
my php that is running is PHP 8.3.13.  I am only trying to run one php.

**Reply by SMillerDev** [reply]
You can always `brew services stop php@8.1`

**Reply by akagumo** [reply]
thank you!  is it safe to uninstall php@8.1? it won't affect the 8.3 install?

**Reply by SMillerDev** [reply]
No, it will not affect other PHP versions.

**Comment by akagumo** [comment]
my info.php in browser shows PHP Version 8.1.30
but terminal php -v shows PHP 8.3.13
whatever I try in brew doesn't fix this, how do I fix it so the system is using 8.3?

**brew link --overwrite --force php@8.3
Warning: Already linked: /usr/local/Cellar/php/8.3.13_1
To relink, run:
  brew unlink php && brew link php
x$ brew unlink php && brew link php
Unlinking /usr/local/Cellar/php/8.3.13_1... 24 symlinks removed.
Linking /usr/local/Cellar/php/8.3.13_1... 24 symlinks created.
x$ brew services restart php@8.3
Stopping `php`... (might take a while)
==> Successfully stopped `php` (label: homebrew.mxcl.php)
==> Successfully started `php` (label: homebrew.mxcl.php)
x$ brew services
Name    Status  User File
httpd   started dp   ~/Library/LaunchAgents/homebrew.mxcl.httpd.plist
php     started dp   ~/Library/LaunchAgents/homebrew.mxcl.php.plist
php@8.0 none         
php@8.1 none         
unbound none         
x$ php -v
PHP 8.3.13**

**Reply by SMillerDev** [reply]
Whatever you're using to serve the website to the browser is probably configured to connect to the PHP 8.1 socket.

**Reply by akagumo** [reply]
I am using brew. is there are way to fix this?

**Reply by gromgit** [reply]
@SMillerDev is talking about the website application that's written in PHP. You need to check your PHP code to make sure it's not using the v8.1 socket.

Also check the web server configuration for your website, the v8.1 socket may also be used there.

**Reply by akagumo** [reply]
oh, sorry I didn't clarify.  I am doing web development on my computer. so I am running local web services and the web site is running locally.  so I ran info.php which is phpinfo () locally and it shows PHP Version 8.1.30 even though brew shows 8.3.13

**Reply by gromgit** [reply]
If you're running `info.php` through your web server, then you need to check your web server configuration. Nothing you've written so far indicates that this is a Homebrew issue.
